spec/e2e/endtoend.spec.js

```diff
@@ -54,34 +71,82 @@ describe('Cordova create and build', function () {
         expect(fs.existsSync(projectFolder)).toBe(true);
     });
 
-    it('spec.2 should build project', function () {
-        shell.exec(buildScriptPath, {silent: true});
-        var packages = shell.ls(appPackagesFolder);
-        expect(packages.filter(function (file) { return file.match(/.*Phone.*\.appx.*/); }).length).toBe(1);
-        expect(packages.filter(function (file) { return file.match(/.*Windows.*\.appx.*/); }).length).toBe(1);
+    // default
+
+    it('spec.2 should build default (win10) project', function () {
```

Comment:
I wonder if this should already be spec.2a?

Reply:
doesn't make a difference really, but I think all the others start with `a` as well, so might as well do this here